plinth/modules/mediawiki/__init__.py

```diff
@@ -40,12 +40,12 @@
       'edited website. You can use MediaWiki to host a wiki-like website, '
       'take notes or collaborate with friends on projects.'),
     _('This MediaWiki instance comes with a randomly generated administrator '
-      'password. You can set a new password in the Configuration section and '
-      'login using the "admin" account. You can then create more user '
+      'password. You can set a new password in the configuration section and '
+      'log in using the "admin" account. You can then create more user '
```

Comment:
`login` is a legitimate word in the current web lingo. I believe its usage as a verb here is correct.

Reply:
What happens is a larger percentage scoff at its use, and errors are introduced in translation for languages where the distinction is even more important.

https://dictionary.cambridge.org/dictionary/english/log-on?q=log+in https://dictionary.cambridge.org/dictionary/english/log-in-on https://www.merriam-webster.com/dictionary/log in?src=search-dict-box https://en.oxforddictionaries.com/definition/user_log-in
It is at most tolerated www.dictionary.com/browse/log--in?s=t#17 vs http://www.dictionary.com/browse/login?s=t